### ISSUE TITLE
'move' animation not working in 0.8.x

### ISSUE THREAD

**Issue by scooper4711** [Feu-Secret/Tokenmagic#128]
Feel free to delete if you're still working on 0.8.x compatibility. The following macro works in 0.7, but not in the latest 0.8.x beta
```
let params =
[{
    filterType: "field",
    filterId: "myHexaField",
    shieldType: 2,
    gridPadding: 2,
    color: 0x4b73aa,
    time: 0,
    blend: 3,
    intensity: 1,
    lightAlpha: 0.5,
    lightSize: 0.5,
    scale: 0.5,
    radius: 0.5,
    chromatic: false,
    alphaDiscard: true,
    zOrder: 5000,
    animated :
    {
      time : 
      { 
        active: true, 
        speed: 0.0015, 
        animType: "move" 
      }
    }
}];

TokenMagic.addUpdateFiltersOnSelected(params);
```

The Hex Field appears, but the animation doesn't work.
This also holds true for the '24 - T02 - Fire Shield' macro.

**Comment by scooper4711** [comment]
```
TypeError: Cannot read property 'placeables' of undefined
    at tokenmagic.js:1554
    at Function._call (eval at <anonymous> (libWrapper-errors.js:249), <anonymous>:4:14)
    at Function.call (foundry.js:179)
    at Canvas.draw (foundry.js:17382)
    at async Scene.view (foundry.js:15376)
    at async Game.initializeCanvas (foundry.js:4413)
    at async Game.setupGame (foundry.js:4292)
    at async Game._initializeGameView (foundry.js:5112)
    at async Game.initialize (foundry.js:4197)
```

**Comment by Feu-Secret** [comment]
Hi thanks for the report. This is a part of the compatibility update, which is fixed in TMFX 0.5.1
